administration/backpressure.md

```diff
@@ -1,5 +1,7 @@
 # Backpressure
 
+<img referrerpolicy="no-referrer-when-downgrade" src="https://static.scarf.sh/a.png?x-pxid=63e37cfe-9ce3-4a18-933a-76b9198958c1" />
```

Comment:
It may be helpful to add HTML comments on the lines preceding these images to state what they're being used for, so that we can both be transparent, and so we can avoid people from trying to strip these from the files.

Reply:
it makes sense. I'm adding it to my ToDo. Thanks for the feedback @cnorris-cs!